src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -45,12 +44,18 @@ import ComposerWithSuggestions from './ComposerWithSuggestions';
 import type {ComposerWithSuggestionsProps} from './ComposerWithSuggestions/ComposerWithSuggestions';
 import SendButton from './SendButton';
 
+// TODO: move this to composer?!
```

Comment:
Do we need to do this?

Reply:
oh so sorry I forgot that in the code - let me quickly refactor that type, hold on

Reply:
let me clean that up in a new PR to not bloat this one!

Reply:
Ok sounds good. Let's create a GH issue for that then and then remove this comment. Otherwise these TODO comments have a tendency to stick around forever 😅